code/game/gamemodes/cult/cult_datums.dm

```diff
@@ -86,20 +86,21 @@
 	entity_title1 = "The Silent One"
 	entity_title2 = "The One Who Beckons"
 	entity_title3 = "The Ferryman of Oblivion"
+	entity_icon_state = "legion"
 
 	cult_wall_icon_state = "deathcult"
 	cult_floor_icon_state = "carpet-broken"
 
 	artificer_name = "Boneshaper"
 
 	behemoth_name = "Draugr"
-	behemoth_icon_state = "horror"
+	behemoth_icon_state = "Draugr"
```

Comment:
are these deffo capitalised in the file? I've come across several sprite issues lately due to caps inconsistency.